dockstore-yaml-proposals/xena/.dockstore.yml

```diff
@@ -17,32 +18,60 @@ service:
      - stop.sh
 
   scripts:
-    # The next 5 scripts are always executed in the order listed here.
-    presprovision: # Before any provisioning
-    prestart: # After provisioning, before starting the service
-    start: stand_up.sh # Starts the service
-    poststart: # After the service has started
-    postprovision: load.sh # After the service has started. For multiple provisionings, e.g., the next day
+    # The scripts preprovision, prestart, start, poststart, and postprovision will be executed in order.
+    # The only script required to have a value is start.
+    
+    # Before any provisioning
+    preprovision: 
+    
+    # After provisioning, before starting the service
+    prestart: 
+     
+    # Starts the service
+    start: bash stand_up.sh 
+    
+    # After the service has started
+    poststart: 
+    
+    # After the service has started. For multiple provisionings, e.g., the user may decide to add more data a week later.
+    postprovision: bash load.sh 
 
-    port: port.sh # Returns the port the service is exposing.
-    healthcheck: # Returns
-    stop: stop.sh # Stops the service
+    # Returns the port the service is exposing. Can be used by a client to know what port to proxy
+    port: bash port.sh
```

Comment:
Idle thought: Bash or shebang? https://www.tldp.org/LDP/abs/html/sha-bang.html
I noticed that the deleted version doesn't have bash in front of scripts, but maybe that makes sense if we can only ensure bash is available

Reply:
When testing it by running it from my Dockstore CLI prototype, it wasn't working on my Ubuntu laptop (but it did work on my Mac laptop), even with sha-bang. This using Java's `Runtime.exec`.  But I was going through several iterations, so maybe I made some fixable mistake.

I'll experiment some more to see if I can't get it working, as you're right, it would be better not to have bash here.

Reply:
Looks like I should have `#!/usr/bin/env bash` instead of `#!/bin/bash` in my scripts, or at least the ones that require bash. See [this](https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang)

Reply:
I guess it depends on what is available by Leonardo. 
I think the Dockstore CLI will always *knock on wood* have Bash

Reply:
Fixed in an upcoming push.